client-js/app/elements/labrad-registry.html

```diff
@@ -8,15 +8,24 @@
     :host {
       display: block;
     }
-    #reg-menu {
+
+    .z-container {
+      background: white;
+      height: 100%;
+      margin: 0px;
+    }
+
+    .reg-menu {
```

Comment:
Since these are inside a DOM module, the rules only apply to this tree (due to the same style encapsulation that bit us with paper-material). So no need to worry about making these styles reusable.